Tasks/PublishCodeCoverageResultsV1/task.json

```diff
@@ -2,7 +2,7 @@
   "id": "2A7EBC54-C13E-490E-81A5-D7561AB7CD97",
   "name": "PublishCodeCoverageResults",
   "friendlyName": "Publish code coverage results",
-  "description": "Publish Cobertura or JaCoCo code coverage results from a build",
+  "description": "[DEPRECATION WARNING. Users are recommended to switch to version 2.235.0.] Publish Cobertura or JaCoCo code coverage results from a build",
```

Comment:
I think the exact minor version would keep on changing in the future. Should we just say , Switch to Version 2.

Reply:
+1
switch to major version 2